dnsping.py

```diff
@@ -219,6 +215,19 @@ def main():
         else:
             i += 1
 
+        if force_miss:
```

Comment:
This code is very obviously mirroring https://github.com/farrokhi/dnsdiag/blob/94d6e48/util/dns.py#L102-L111